### PR TITLE
[CFX-590] [CFX-539] Make Notebook custom environments to be compatible with custom models

### PR DESCRIPTION
**Pull request by lyapun**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

Make Notebook custom environments to be compatible with Custom Models

### CODE REVIEW

File: public_dropin_notebook_environments/python311_notebook/requirements.txt [hunk 1]
```diff
@@ -33,8 +35,7 @@ spacy==3.7.2
 statsmodels==0.14.1
 tqdm==4.64.1
 umap-learn==0.5.3
+uwsgi
```
Comment (c-h-russell-walker) [inline public_dropin_notebook_environments/python311_notebook/requirements.txt:38]: Intentionally not pinned?

**Review by c-h-russell-walker — APPROVED** [review]
Looks good 👍 

I assume you intentionally didn't pin `uwsgi`

Only thing I'd say if it's at all important os that the different, newly added start server files have a copyright year of 2021 versus 2024 🤷

**Comment by yakov-g** [comment]
What is the plan on releasing these environments. 
We plan to add validation, that custom model can only be created with the env supporting custom models: useCase in the env_info.json should have `customModel` value.

**Review by yakov-g — APPROVED** [review]
Approve while it is not released.

File: public_dropin_notebook_environments/python311_notebook/requirements.txt (hunk 1 above)
Comment (lyapun) [reply public_dropin_notebook_environments/python311_notebook/requirements.txt:38]: yes, like in custom models custom envs

**Comment by lyapun** [comment]
@yakov-g, added "customModel" to the list of use cases. Is it good to be released now?

**Review by c-h-russell-walker — APPROVED** [review]

**Pull request merged**